server/src/metadata.rs

```diff
@@ -48,6 +48,26 @@ pub mod entities_metadata_query {
         entities: Vec<EntityMetadata>,
     }
 
+    #[derive(Serialize)]
+    enum CreatorType {
+        Person,
+        // Should we support this type too?
+        // Organization,
+    }
+
+    #[derive(Serialize)]
+    #[serde(rename_all = "camelCase")]
+    struct Creator {
+        #[serde(rename = "type")]
+        creator_type: CreatorType,
+        id: Option<String>,
```

Comment:
```suggestion
        id: String,
```

We will never have `None` here.